modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Environment.scala

```diff
@@ -169,24 +172,31 @@ object Environment {
       good <- sinkGood
       bad <- sinkBad
       pii <- sinkPii.sequence
-      http <- Clients.mkHttp(ec)
+      http <- Clients.mkHttp(ec = ec)
       clts <- clients.map(Clients.init(http, _))
       igluClient <- IgluClient.parseDefault[F](parsedConfigs.igluJson).resource
       metrics <- Resource.eval(Metrics.build[F](blocker, file.monitoring.metrics))
       metadata <- Resource.eval(metadataReporter[F](file, processor.artifact, http))
       assets = parsedConfigs.enrichmentConfigs.flatMap(_.filesToCache)
+      (remoteAdapterHttpClient, remoteAdapters) <- prepareRemoteAdapters[F](file.remoteAdapters, ec)
+      adapterRegistry = new AdapterRegistry(remoteAdapters)
       sem <- Resource.eval(Semaphore(1L))
       assetsState <- Resource.eval(Assets.State.make[F](blocker, sem, clts, assets))
-      enrichments <- Enrichments.make[F](parsedConfigs.enrichmentConfigs, BlockerF.ofBlocker(blocker))
+      enrichments <- {
+        implicit val C: Http4sClient[F] = http
+        Enrichments.make[F](parsedConfigs.enrichmentConfigs, BlockerF.ofBlocker(blocker))
+      }
```

Comment:
Is there a reason why you use `HttpClient` implicitly everywhere instead of just passing it as a regular parameter? It could ease the reading of the code